edxadapt/static/js/src/edxadapt.js

```diff
@@ -0,0 +1,82 @@
+/* Javascript for EdxAdaptXBlock. */
+function EdxAdaptXBlock(runtime, element) {
+
+    var studentId = '{{anonymous_student_id}}';
+    var apiBaseUrl = '{{edx_adapt_api_url}}';
+    var courseId = '{{course_id}}';
+    var params = {{params}};
+    var skills = {{skills}};
```

Comment:
Actually could these variable values be passed through and read from the HTML as "data-" attributes so that this JS file can be minified and cached?